### PR TITLE
Create contributor-interview-snor42.md

### PR DESCRIPTION
**Pull request by l-delin**

### CODE REVIEW

**Comment by l-delin** [comment]
This interview is ready for review :-) If possible, could it be posted for mid-June? Thanks!

**Review by mflasquin — APPROVED** [review]

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+aliases: ["/news/contributor-interview-snor42"]
+title: "Freelance developper and Translator: meet SNor42"
```
Comment (coffrep) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md]:
```suggestion
title: "Freelance Developer and Translator: Meet SNor42"
```

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+subtitle: "Discover the faces behind translations"
+date: 2023-12-06
```
Comment (coffrep) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md:6]:
```suggestion
date: 2023-06-13
```

**Review by coffrep — DISMISSED** [review]
Thanks @l-delin for the interview, super nice. Just added two suggestions. The date from what I saw in other posts is a Year/month/day format, hope this helps.

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md (hunk 2 above)
Comment (kpodemski) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md]:
```suggestion
date: 2023-06-20
```

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+- community
+
+- contribution
+---
+**Have you ever wondered who’s hiding behind a Crowdin username? It’s time to find out! Contributor interviews are a great way of getting to know the people who participate in the PrestaShop project.**
+
+**For the second interview of 2023, we interview Norbert, aka [SNor42](https://crowdin.com/profile/snor42).**
+
+<img src="https://github.com/PrestaShop/prestashop.github.io/assets/113511191/5a649c1c-bb2c-465a-9e49-e1e4f737da67" width="384" height="512">
```
Comment (kpodemski) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md:21]: it's not the best way to use photos in the article, I'll fix it after the merge

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+**Ranked 29th on the [top translator page](https://translators.prestashop.com/), Norbert is a translator and proofreader on the Hungarian project.**
+
+<div style="clear:both"></div>
```
Comment (kpodemski) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md]:
```suggestion
```

this shouldn't be needed, every time you use enter it should be all good

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md (hunk 1 above)
Comment (kpodemski) [reply content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md]: I think it should be a "Developer", right?

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md (hunk 3 above)
Comment (l-delin) [reply content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md:21]: Still not too sure how to add pictures, I'm experimenting everything at this point 🤪

**Comment by l-delin** [comment]
Thanks @kpodemski 😄

**Comment by l-delin** [comment]
Can we get this published, please? Thanks :-)

File: content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+aliases: ["/news/contributor-interview-snor42"]
+title: "Freelance Developer and Translator: Meet SNor42"
+subtitle: "Discover the faces behind translations"
+date: 2023-06-20
```
Comment (kpodemski) [inline content/news/posts/2023/content/news/posts/2023/contributor-interview-snor42.md]:
```suggestion
date: 2023-06-23
```

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @l-delin 

merged, I fixed the image in a separate commit :) as you can see I put it locally to a correct folder, and used a relative path

**Comment by l-delin** [comment]
Thank youuuu @kpodemski :-)

**Pull request merged**